spec/mongo/collection_spec.rb

```diff
@@ -1710,6 +1710,16 @@
         }.to raise_exception(Mongo::Error::OperationFailure)
       end
     end
+
+    context 'when write_concern is provided', if: find_command_enabled? && standalone? do
+
+      it 'raises OperationFailure when w=2' do
```

Comment:
You can just say 'uses the write concern' because the test verifies that the write concern is set. The fact that it raises the OperationFailure is proof of this, not what we are actually testing.